src/app/application/components/app-collection/AppCollection.vue

```diff
@@ -225,7 +225,8 @@ const click = (e: MouseEvent) => {
 </script>
 
 <style lang="scss" scoped>
-.app-collection :deep(td:first-of-type > a) {
+.app-collection :deep(td:first-child),
+.app-collection :deep(td:first-child > *) {
```

Comment:
Why this change? I'm not sure if we ever had a table that didn't link somewhere we would want it to use the bold style for the first cell?

Reply:
That style is meant to highlight the name (i.e. the resource-identifying feature) first. It shouldn’t be a style specific to links in the first place.